src/common/init.cpp

```diff
@@ -46,22 +48,23 @@ void PelotonInit::Initialize() {
   gc::GCManagerFactory::GetInstance().StartGC();
 
   // start index tuner
-  if (FLAGS_index_tuner == true) {
+  if (Config::GET_BOOL("index_tuner")) {
```

Comment:
This is what I don't want to happen. We don't people to pass an arbirtraty string to get a parameter because we have no way of checking this at compile time. Can you define a new `enum class` that has the list of parameters?